mpsl/CMakeLists.txt

```diff
@@ -19,4 +19,8 @@ zephyr_include_directories(include)
 zephyr_include_directories(include/protocol)
 zephyr_link_libraries(${MPSL_LIB_DIR}/libmpsl.a)
 
+if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/fem)
```

Comment:
could you please wait with this PR until you are ready to deploy the libs ?

There is no reason to expose this to NCS users if the libs are not ready to be deployed.

Reply:
Ok, I will decline this PR and put the change with the deploy.